### PR TITLE
[6.13.z] Add test coverage for BZ:1931489

### PR DESCRIPTION
**Pull request by Satellite-QE**
Cherrypick of PR: https://github.com/SatelliteQE/robottelo/pull/15035

### Problem Statement
Missing coverage for BZ:1931489, where we verified timeout to kill/execution_timeout_interval setting is honored by long running ansible-playbook job

### Solution
Add test coverage for BZ:1931489, to verify long running ansible-playbook job along with timeout to kill/execution_timeout_interval setting

### Dependent PRs:
https://github.com/SatelliteQE/nailgun/pull/1139

### CODE REVIEW

**Comment by Satellite-QE** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill
nailgun: 1139

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6956
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill --external-logging
Test Result : ================= 1 failed, 13 warnings in 1485.93s (0:24:45) ==================
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill
nailgun: 1139

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6981
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill --external-logging
Test Result : ================= 1 failed, 13 warnings in 1327.58s (0:22:07) ==================
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7367
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill --external-logging
Test Result : ================= 1 failed, 13 warnings in 1436.02s (0:23:56) ==================
```

File: tests/foreman/api/test_ansible.py [hunk 1]
```diff
@@ -349,3 +349,72 @@ def test_positive_ansible_localhost_job_on_host(
...
+        result = target_sat.api.JobInvocation(id=job['id']).read()
+        assert result.pending == 0
+        assert result.failed == 1
+        assert result.status_label == 'failed'
+
+        result = target_sat.api.JobInvocation(id=job['id']).outputs()['outputs'][0]['output']
+        termination_msg = 'Timeout for execution passed, stopping the job'
+        assert [i['output'] for i in result if i['output'] == termination_msg]
+        assert [i['output'] for i in result if i['output'] == 'StandardError: Job execution failed']
+        assert [i['output'] for i in result if i['output'] == 'Exit status: 120']
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_ansible.py:420]:
```suggestion
        assert [i['output'] for i in result if i['output'] == 'Exit status: 2']
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7371
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill --external-logging
Test Result : ================= 1 passed, 13 warnings in 1368.12s (0:22:48) ==================
```

**Pull request merged**